lib/Stats.js

```diff
@@ -974,18 +974,15 @@ class Stats {
 		}
 		if (typeof obj.builtAt === "number") {
 			const builtAtDate = new Date(obj.builtAt);
-			let timeZone = null;
+			let timeZone;
```

Comment:
```suggestion
			let timeZone = undefined;
```

Could you set it to undefined, to make it clear that this is intended?

Reply:
009e47c